packages/lesswrong/server/migrations/2020-05-06-forceSecureImageLinks.ts

```diff
@@ -89,6 +89,7 @@ registerMigration({
   action: async () => {
     await forEachDocumentBatchInCollection({
       collection: Posts,
+      batchSize: 500,
```

Comment:
doublechecking this file meant to get changed?

Reply:
Yeah, it's a small bugfix that seemed nice to have. I also want us to run this migration soon, so I was experimenting with it while I was doing a bunch of other migration-related work.